src/storageTypes.ts

```diff
@@ -13,6 +13,7 @@ export interface Card {
     id: string;
     timeCreated: number;
     source: string;
+    view: boolean;
```

Comment:
Earlier downloads of the extension will not have the `view` property in storage, so best to make it optional here or update storage formatting in the backend upon extension update.